mypy.ini

```diff
@@ -1,6 +1,217 @@
+
+[mypy-lxml.*]
+ignore_missing_imports = True
+
+[mypy-matplotlib.*]
+ignore_missing_imports = True
+
+[mypy-numexpr]
+ignore_missing_imports = True
+
+[mypy-numpy.*]
+ignore_missing_imports = True
+
+[mypy-openpyxl.*]
+ignore_missing_imports = True
+
+[mypy-pandas._libs.algos]
+ignore_missing_imports = True
+
+[mypy-pandas._libs.groupby]
+ignore_missing_imports = True
+
+[mypy-pandas._libs.hashing]
+ignore_missing_imports = True
+
+[mypy-pandas._libs.hashtable]
+ignore_missing_imports = True
+
+[mypy-pandas._libs.index]
+ignore_missing_imports = True
+
+[mypy-pandas._libs.indexing]
+ignore_missing_imports = True
+
+[mypy-pandas._libs.internals]
+ignore_missing_imports = True
+
+[mypy-pandas._libs.interval]
+ignore_missing_imports = True
+
+[mypy-pandas._libs.join]
+ignore_missing_imports = True
+
+[mypy-pandas._libs.json]
+ignore_missing_imports = True
+
+[mypy-pandas._libs.lib]
+ignore_missing_imports = True
+
+[mypy-pandas._libs.missing]
+ignore_missing_imports = True
+
+[mypy-pandas._libs.ops]
+ignore_missing_imports = True
+
+[mypy-pandas._libs.parsers]
+ignore_missing_imports = True
+
+[mypy-pandas._libs.properties]
+ignore_missing_imports = True
+
+[mypy-pandas._libs.reduction]
+ignore_missing_imports = True
+
+[mypy-pandas._libs.reshape]
+ignore_missing_imports = True
+
+[mypy-pandas._libs.sparse]
+ignore_missing_imports = True
+
+[mypy-pandas._libs.testing]
+ignore_missing_imports = True
+
+[mypy-pandas._libs.tslib]
+ignore_missing_imports = True
+
+[mypy-pandas._libs.tslibs.c_timestamp]
+ignore_missing_imports = True
+
+[mypy-pandas._libs.tslibs.ccalendar]
+ignore_missing_imports = True
+
+[mypy-pandas._libs.tslibs.conversion]
+ignore_missing_imports = True
+
+[mypy-pandas._libs.tslibs.fields]
+ignore_missing_imports = True
+
+[mypy-pandas._libs.tslibs.frequencies]
+ignore_missing_imports = True
+
+[mypy-pandas._libs.tslibs.nattype]
+ignore_missing_imports = True
+
+[mypy-pandas._libs.tslibs.np_datetime]
+ignore_missing_imports = True
+
+[mypy-pandas._libs.tslibs.offsets]
+ignore_missing_imports = True
+
+[mypy-pandas._libs.tslibs.parsing]
+ignore_missing_imports = True
+
+[mypy-pandas._libs.tslibs.period]
+ignore_missing_imports = True
+
+[mypy-pandas._libs.tslibs.resolution]
+ignore_missing_imports = True
+
+[mypy-pandas._libs.tslibs.strptime]
+ignore_missing_imports = True
+
+[mypy-pandas._libs.tslibs.timestamps]
+ignore_missing_imports = True
+
+[mypy-pandas._libs.tslibs.timedeltas]
+ignore_missing_imports = True
+
+[mypy-pandas._libs.tslibs.timezones]
+ignore_missing_imports = True
+
+[mypy-pandas._libs.tslibs.tzconversion]
+ignore_missing_imports = True
+
+[mypy-pandas._libs.window]
+ignore_missing_imports = True
+
+[mypy-pandas._libs.writers]
+ignore_missing_imports = True
+
+[mypy-pandas.core.arrays.sparse.array]
+ignore_missing_imports = True
+
+[mypy-pandas.io.msgpack._packer]
+ignore_missing_imports = True
+
+[mypy-pandas.io.msgpack._unpacker]
+ignore_missing_imports = True
+
+[mypy-pandas.io.sas._sas]
```

Comment:
why do the cython live have to be here?

Reply:
These throw errors like `error: No library stub file for module ‘<the_pyx_file>’` - I imagine the annotations don't find anything that gets imported.

I *think* the only way around this may be to add stub files for the internally used aspects of the .pyx files but haven't found anything definitive yet. Also asked if anyone had experience with this on the typing Gitter but no luck yet. Stub files might not be a terrible fallback if that's all that works